source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
All folder names shall always start with lower case letters. Files named after the containing class are named with an initial upper case letter and the rest with a lower case letter. Does this make sense?